src/test/java/org/zeromq/TestRapidOpenCloseSocket.java

```diff
@@ -0,0 +1,75 @@
+package org.zeromq;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import org.zeromq.ZMQ.Socket;
+
+// For issue: https://github.com/zeromq/jeromq/issues/198
+public class TestRapidOpenCloseSocket
+{
+  @Test(timeout = 10000)
```

Comment:
Definitely put @Ignore. This is a failing test as there is no fix available as of now, and this is a situation that is present since... a loooong time, so it is mostly to document behaviour so we can have a baseline.
If this is merged, it will provoke systematic fail on every subsequent PRs, preventing a better visibility of the status of the lib.
When this is fixed, let's just remove the annotation and let's make a toast for that :)

Reply:
Sounds reasonable enough -- done.